spec.bs

```diff
@@ -325,24 +326,33 @@ dictionary AuctionAdInterestGroupKey {
 };
 </xmp>
 
+<div algorithm>
+
 The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps

```

Comment:
do we want to mention explainer's "invoking navigator.leaveAdInterestGroup() from inside an ad that is being targeted at a particular interest group will cause the browser to leave that group, irrespective of permission policies" somehow?

Reply:
I think the TODO 18 lines above does that already?

Reply:
ah yes, I remember I've seen that :)